source/guides/account/how-to-use-ledger-hardware-wallet.rst

```diff
@@ -0,0 +1,307 @@
+
+2. Review your Symbol address, then press the right button. In this tutorial, the Symbol address is shown below.
+
+.. figure:: ../../resources/images/screenshots/ledger-guide-7.png
+  :align: center
+  :width: 100%
+  :class: with-shadow
+  :target: /_images/ledger-guide-7.png
+
+3. Press both left and right buttons simultaneously to approve.
+
+.. figure:: ../../resources/images/screenshots/ledger-guide-8.png
+  :align: center
+  :width: 100%
+  :class: with-shadow
+  :target: /_images/ledger-guide-8.png
+
+*************************************************
+Step 5: Send and receive XYM in the Ledger device
+*************************************************
+
+After completing the steps mentioned above, navigate to the Symbol Wallet dashboard by clicking **Home** in the left side panel.
+
+The dashboard shows all the transactions coming in and out of your Ledger device, your account balance, assets and block information.
+
+.. figure:: ../../resources/images/screenshots/ledger-guide-9.png
+  :align: center
+  :width: 100%
+  :class: with-shadow
+  :target: /_images/ledger-guide-9.png
+
+
+**IMPORTANT**: When initializing a transaction in your Symbol Wallet, always verify if the data (e.g. Symbol address, XYM amount) matches with the corresponding data displayed on your Ledger device. If the two data match, you can proceed with the transaction; otherwise, you should NOT continue.
+
+******************
+How to receive XYM
+******************
+
+1. Click **Accounts** in the left side panel.
+
+.. figure:: ../../resources/images/screenshots/ledger-guide-10.png
+  :align: center
+  :width: 100%
+  :class: with-shadow
+  :target: /_images/ledger-guide-10.png
+
+2. Click the **Show on Ledger** button to verify your Symbol address. If the address in your Symbol Wallet matches with the address in your Ledger device, you can rest assured that it is true and genuine.
+
+3. Once verified, you can now share your Symbol address with the sender.
+
+***************
+How to send XYM
+***************
+
+1. Click **Transfer**.
+
+2. Fill out the required fields.
+
+- **To** - the receiver’s XYM address.
+- **Amount of XYM** - the amount you are sending to the receiver.
+
+**IMPORTANT**: If you are sending XYM to an exchange, you might need to add an unencrypted message. Please read carefully the instructions provided by the exchange.
+
+3. Click the **Send** button. Note that there will be a fee for every transaction made.
+
+.. figure:: ../../resources/images/screenshots/ledger-guide-11.png
+  :align: center
+  :width: 100%
+  :class: with-shadow
+  :target: /_images/ledger-guide-11.png
+
+.. figure:: ../../resources/images/screenshots/ledger-guide-12.png
+  :align: center
+  :width: 100%
+  :class: with-shadow
+  :target: /_images/ledger-guide-12.png
+
+4. Review each transaction detail in your Ledger device and press the right button to continue.
+
+* Recipient’s Symbol address
+
+.. figure:: ../../resources/images/screenshots/ledger-guide-13.png
```

Comment:
It looks like this should be image 14 for the address and image 13 is the transaction type?